### PR TITLE
Top fixes

### PR DESCRIPTION
**Pull request by soupdiver**
See #76

### CODE REVIEW

**Comment by requilence** [comment]
What about other things we talked
- Because of sorting, using the `Load` we will not likely see the short-lived high-spiked process
- Querying every second is too frequently 

I propose: 
- Add `Load1`(avg1) metric and use it for sorting
- Query every 5 seconds instead of 1 second. Because even some proc that lived 5 sec made a 100% CPU load it will result in `avg1=8%` so the user will likely ignore this

**Comment by soupdiver** [comment]
> What about other things we talked

Not mentioned in the issue

**Comment by soupdiver** [comment]
> Add Load1(avg1) metric and use it for sorting

done

**Comment by soupdiver** [comment]
> Query every 5 seconds instead of 1 second.

Done

File: pkg/monitoring/top/top.go [hunk 1]
```diff
@@ -91,6 +94,7 @@ func (t *Top) startMeasureProcessLoad(interval time.Duration) {
 					PID:        p.PID,
 					Command:    p.Command,
 					Identifier: p.Name,
+					Load1:      ring.New(60),
```
Comment (requilence) [inline pkg/monitoring/top/top.go]: you should divide ring.New arg with `interval`

Comment (soupdiver) [reply pkg/monitoring/top/top.go]: that is correct 👍 
done

**Comment by requilence** [comment]
@soupdiver maybe it is a good idea to round float with 2 decimal place? 
```
"cpu_analyser.top":[{"name":"stress","pid":19685,"command":"stress --cpu 2 --timeout 600","load":32.00714496243167,"load1":27.151941315673664,"load5":27.718735561029046,"load15":27.76807674163746
```

**Pull request merged**